webapi/src/main/scala/org/knora/webapi/routing/Proxy.scala

```diff
@@ -56,18 +56,19 @@ trait Proxy {
     private def updateRequest(uri: Uri, unmatchedPath: Uri.Path, updateUri: (Uri, Uri.Path, RequestContext) => Uri): RequestContext => HttpRequest =
         ctx => ctx.request.copy(
             uri = updateUri(uri, unmatchedPath, ctx),
-            headers = stripHostHeader(ctx.request.headers)
+            headers = stripHostHeader(ctx.request.headers.toList)
         )
 
     private def proxyRequest(updateRequest: RequestContext => HttpRequest)(implicit system: ActorSystem): Route =
         ctx => IO(Http)(system) tell(updateRequest(ctx), ctx.responder)
 
-    private def stripHostHeader(headers: List[HttpHeader]) = {
-        println(s"headers before strip: ${headers.toString}")
-        val strippedHeaders = headers filterNot (header => header is (HttpHeaders.Host.lowercaseName))
+    private def stripHostHeader(headersList: List[HttpHeader]) = {
```

Comment:
If you need debugging output, please use the Akka system logger instead of println.